### ISSUE TITLE
Archive - Pagination, move to the new pagination class

### ISSUE THREAD

**Issue by jenstornell** [jenstornell/kirby-seo#16]

**Comment by Thiousi** [comment]
This seems like it's been added already?

**Comment by jenstornell** [comment]
Yes, that's true but now I've developed a better pagination class and I'll think there would be better to move to that class. The code would be better separated.

But indeed, this issue then needs to be renamed.
